system/file_windows.go

```diff
@@ -0,0 +1,31 @@
+package system
+
+import (
+	"fmt"
+	"os"
+)
+
+// Mode returns golang file mode. This method is cross-platform, but we are keeping unix-specific implementation
+// in file_unix.go for backwards compatibility.
+func (f *DefFile) Mode() (string, error) {
+	if err := f.setup(); err != nil {
+		return "", err
+	}
+
+	fi, err := os.Lstat(f.realPath)
+	if err != nil {
+		return "", err
+	}
+
+	mode := fmt.Sprintf("%04o", (fi.Mode() & 07777))
```

Comment:
Note that `fi.Mode` has a bit different format from `sys.(*syscall.Stat_t).Mode`.